flink-formats/flink-parquet/src/test/java/org/apache/flink/formats/parquet/ParquetColumnarRowInputFormatTest.java

```diff
@@ -294,6 +294,80 @@ void testProjectionReadUnknownField(int rowGroupSize) throws IOException {
                 });
     }
 
+    @ParameterizedTest
+    @MethodSource("parameters")
+    void testProjectionReadUnknownFieldAcrossFiles(int rowGroupSize) throws IOException {
+        int number = 1000;
```

Comment:
What's the intention behind having such a large testdata set? Is it because we iterating over multiple group sizes that are provided through the `parameter()` test data method source? If the `1000` was selected because of that, it would be worth creating a static final field `MAX_ROW_GROUP_SIZE` and use this here as well. That would make it clearer why the value `1000` was selected.